sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -37,6 +37,28 @@ import { UserAgentOptions } from '@azure/core-http';
 import { UserDelegationKeyModel } from '@azure/storage-blob';
 import { WebResource } from '@azure/core-http';
 
+// @public
+export interface AccessControlChangeCounters {
+    changedDirectoriesCount: number;
+    changedFilesCount: number;
+    failedChangesCount: number;
+}
+
+// @public
+export interface AccessControlChangeFailure {
+    errorMessage: string;
+    isDirectory: boolean;
+    name: string;
+}
+
+// @public
+export interface AccessControlChanges {
+    aggregateCounters: AccessControlChangeCounters;
+    batchCounters: AccessControlChangeCounters;
```

Comment:
I don't like how `batchCounters` is an object and `batchFailures` is an array. I might be happier if the `batchCounters` properties were inlined into `AccessControlChanges`. Thoughts?

Reply:
Then we need to inline both `aggregateCounters` and `batchCounters` like:

```javascript
interface AccessControlChanges {
    changedDirectoriesCount: number;
    changedFilesCount: number;
    failedChangesCount: number;
    aggregateChangedDirectoriesCount: number;
    aggregateChangedFilesCount: number;
    aggregateFailedChangesCount: number;
}
```

`counters` in response types need to be inlined too, but should be good.

Reply:
@bterlson 

Does following sounds good to you?

```javascript
interface AccessControlChanges {
    changedDirectoriesCount: number;
    changedFilesCount: number;
    failedChangesCount: number;
    aggregateChangedDirectoriesCount: number;
    aggregateChangedFilesCount: number;
    aggregateFailedChangesCount: number;
}
```

Reply:
@bterlson kindly ping~

Reply:
I guess this isn't significantly better. I leave this to your better judgement (and let's just make sure it's reasonably consistent with what others are doing here).

Reply:
Thanks Brian! .Net chooses to use `AccessControlChanges` wraping `batchCounters` and `aggregateCounters`. I'd like JS do a consistent way. Refer to https://github.com/Azure/azure-sdk-for-net/blob/feature/storage/recursive-acl/sdk/storage/Azure.Storage.Files.DataLake/src/Models/AccessControlChanges.cs